Description: add and run our build script
 Also, use bionic (18.04) as the base image, since xenial lacks
 libayatana-appindicator3-dev.
Author: Amin Bandali <amin.bandali@savoirfairelinux.com>

--- ../docker/Dockerfile-snap.original	2020-11-23 15:51:17.211591055 -0500
+++ ../docker/Dockerfile-snap	2020-11-23 15:52:29.247759416 -0500
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial as builder
+FROM ubuntu:bionic as builder
 
 # Grab dependencies
 RUN apt-get update
@@ -36,7 +36,7 @@
 
 # Multi-stage build, only need the snaps from the builder. Copy them one at a
 # time so they can be cached.
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 COPY --from=builder /snap/core /snap/core
 COPY --from=builder /snap/core18 /snap/core18
 COPY --from=builder /snap/snapcraft /snap/snapcraft
@@ -53,3 +53,7 @@
 ENV SNAP="/snap/snapcraft/current"
 ENV SNAP_NAME="snapcraft"
 ENV SNAP_ARCH="amd64"
+
+ADD scripts/build-package-snap.sh /opt/build-package-snap.sh
+
+CMD ["/opt/build-package-snap.sh"]
